src/test/java/com/google/api/codegen/discogapic/testdata/java/java_simplecompute.v1.json.baseline

```diff
@@ -11874,21 +11890,22 @@ public class AddressClient implements BackgroundResource {
    * <pre><code>
    * try (AddressClient addressClient = AddressClient.create()) {
    *   ProjectRegionAddressName address = ProjectRegionAddressName.of("[PROJECT]", "[REGION]", "[ADDRESS]");
-   *   Operation response = addressClient.deleteAddress(address);
+   *   addressClient.deleteAddressAsync(address).get();
    * }
    * </code></pre>
    *
    * @param address Name of the address resource to delete.
    * @throws com.google.api.gax.rpc.ApiException if the remote call fails
    */
   @BetaApi
-  public final Operation deleteAddress(ProjectRegionAddressName address) {
```

Comment:
Should we leave in the original methods? It would be a little different than what happens in gRPC clients, but it allows us to later flag an API method as LRO without removing the original generated method.